spec/lib/gds_api/whitehall_export_spec.rb

```diff
@@ -36,4 +35,18 @@
       expect(whitehall_adapter.document_export("123")).to have_requested(:get, "#{whitehall_host}/government/admin/export/document/123")
     end
   end
+
+  def stub_publishing_api_has_index_page(lead_organisation, document_type, document_subtypes, page_number, items_on_page)
```

Comment:
Should this be called something like `stub_whitehall_api_has_document_index`?

Reply:
Yes, that seems like a better name.  Have updated.